### PR TITLE
Add support for skip backward and forward

### PR DESCRIPTION
**Pull request by fastred**
#### What's this PR do?

This PR adds support for skipping backward and forward on iOS.

### CODE REVIEW

**Comment by tanguyantoine** [comment]
Can you add example on the README for this feature?

File: index.ios.js [hunk 1]
```diff
@@ -23,8 +23,8 @@ var MusicControl = {
   resetNowPlaying: function(){
     NativeMusicControl.resetNowPlaying()
   },
-  enableControl: function(controlName, bool){
-    NativeMusicControl.enableControl(controlName, bool)
+  enableControl: function(controlName, bool, options){
```
Comment (tanguyantoine) [inline index.ios.js]: `enableControl: function(controlName, bool, options = {}){``

adding default will prevent null exception on Objective-C side no?

File: index.ios.js [hunk 2]
```diff
@@ -23,8 +23,8 @@ var MusicControl = {
...
+    NativeMusicControl.enableControl(controlName, bool, options)
```
Comment (tanguyantoine) [inline index.ios.js]: probably here too, what do you think of `options || {}` ?

**Comment by tanguyantoine** [comment]
Except what i commented all is a good idea

**Comment by fastred** [comment]
@tanguyantoine Good points, thanks! I added two new commits.

**Comment by tanguyantoine** [comment]
You rocks

**Pull request merged**